rfcs/0009-code-review.md

```diff
@@ -0,0 +1,192 @@
+   - This allows them to make any final touch ups or edit the commit
+     message.
+   - Using merge commits ends up with the person pressing the merge button as
+     the author in GitHub and makes the authorship harder to understand.
+ - Cleanup branches after merge. If possible, enable the auto-delete
+   after merge option on repositories.
+ - Prefer to `Squash and merge` commits.
+   - The development history is in the PR if desired, but typically
+     doesn't add much to the main repository.
+ - Force pushing to a repo should be a last resort.
+
+### Principles for reviewees
+ - The title and description should provide all the needed context.
+   - The PR will serve as a record of the change into the future.
+   - Anything important for the reviewer to know (other approaches tried,
+     testing you’ve done, etc) is worth including.
+   - Link to any associated issues or Trello cards.
+   - If the PR is broken apart to make review easier, explain how it was
+     broken apart and include the next steps in the PR description.
+   - List any steps, caveats or gotchas that the reviewer will need to
+     know.
+ - Do your best to address all comments (even just to say "Done" or put
+   in a thumbs up).
+   - Marking the comments as `Resolved` makes it easier to understand
+     what still needs to be done.
+ - Assume the best intention from review comments
+ - Wait for the tests to pass before you merge.
+ - Tend towards smaller changes. They are easier to understand and
+   faster to review.
+
+
+### Principles for reviewers
+ - Try to comment on the PR in as short an interval is possible (<24
+   hours preferably, excluding weekends).
+   - The comment doesn't have to be the review, but can just be a
+     comment letting the reviewee know when you'll be able to review.
+     This gives them the opportunity to find another reviewer if needed.
+   - If you’re not sure how someone wants their code reviewed or how they
+     prefer to receive feedback, talk to them beforehand.
+ - Point out when something is particularly good or clever.
+   - Reviews often focus on the negative so highlight the good as well
+     when you see it.
+ - Be explicit.
+   - Include screenshots or gifs if they help make your point.
+ - Programming is subjective.
+   - Different approaches are valid.
+   - Ask for clarifications if you need any.
+ - Communicate which ideas you feel strongly about and those you don't.
+
+
+### Technical Considerations
+
+#### Logic
+ - Does the code work?
+   - Does it perform its intended function?
+   - Is the logic correct?
+   - Is the proposed UI accessible?
+ - Does this the change introduce tech debt?
+   - Adding tech debt isn't a bad thing, but we need to make sure we understand
+     why we're adding it and how we're going to contain it.
+   - If adding, are there comments or documentation around the debt so future
+     maintainers can understand the issues?
+ - Does this change introduce any security concerns?
```

Comment:
should we ask if it impacts any existing ATO/Security paperworks?

Reply:
Done.